help/user-guide/index.md

```diff
@@ -5,88 +5,130 @@ layout: page
-     - [Assay Type](general-attributes.html#assay-type)
-     - [Technology Type](general-attributes.html#technology-type)
-     - [Organisms](general-attributes.html#organisms)
-     - [Experimentalists](general-attributes.html#experimentalists)
-     - [Sharing](general-attributes.html#sharing)
-     - [Publishing](general-attributes.html#publishing)
-     - [Tags](general-attributes.html#tags)
-     - [Attributions](general-attributes.html#attributions)
-     - [Creators](general-attributes.html#creators)
-     - [SOPs](general-attributes.html#sops)
-     - [Data Files](general-attributes.html#data-files)
-     - [Placeholders](general-attributes.html#placeholders)
-     - [File Templates](general-attributes.html#file-templates)
-     - [Publications](general-attributes.html#publications)
-     - [Experimental assays and Modelling analysis](general-attributes.html#experimental-assays-and-modelling-analysis)
-     - [Events](general-attributes.html#events)
+- [Viewing Project in Experiment View](viewing-project-in-single-page.html)
+- [Specialist user roles and their capabilities](roles.html)
+
+Specialist user roles
+  - [Programme administrator](roles.html#programme-administrator)
+  - [Project administrator](roles.html#project-administrator)
+  - [Asset housekeeper](roles.html#asset-housekeeper)
+  - [Asset gatekeeper](roles.html#asset-gatekeeper)
+
+Capabilities
+  - [Assign people to Project roles](administer-project-members.html#assign-people-to-project-roles)
+  - [Flag when a person becomes inactive in a Project](administer-project-members.html#flag-when-a-person-becomes-inactive-in-a-project)
+  - [Create Organisms](adding-admin-items.html#creating-organisms)
+  - [Create Profiles](adding-admin-items.html#creating-profiles)
+  - [Create Institutions](adding-admin-items.html#creating-institutions)
+  - [Add and remove people from a Project](administer-project-members.html#add-and-remove-people-from-a-project)
+
+## Experiments
+- [Generating the ISA structure](generating-the-isa-structure.html)
+  - [Creating an Investigation](generating-the-isa-structure.html#creating-an-investigation)
+  - [Creating a Study](generating-the-isa-structure.html#creating-a-study)
+  - [Creating an Experimental assay](generating-the-isa-structure.html#creating-an-experimental-assay)
+  - [Creating a Modelling analysis](generating-the-isa-structure.html#creating-a-modelling-analysis)
+- [ISA Overview Graph](isa-overview.html)
+
+- [Designing an ISA-JSON compliant experiment](designing-experiments-isajson-compliant.html)
+  - [Creating an ISA Investigation](designing-experiments-isajson-compliant.html#1-creating-an-isa-investigation)
+  - [Creating an ISA Study](designing-experiments-isajson-compliant.html#2-creating-an-isa-study)
+  - [Creating an ISA Assay](designing-experiments-isajson-compliant.html#5-creating-an-assay-stream)
+- [ISA-JSON compliant experiment](isa-json-compliant-experiment.html)
```

Comment:
I suggest to put "ISA-JSON compliant experiment" before "Designing an ISA-JSON compliant experiment"